ansible/roles/airflow/tasks/af-container.yml

```diff
@@ -0,0 +1,36 @@
+---
+- file: path=/srv/{{ dockname }} state=directory mode=0750 owner={{ login }} group={{ login }}
+- stat: path=/srv/{{ dockname }}/airflow.cfg
+  register: afcfg
+- command: cp -a /srv/etc/af-share-{{ airflow_tag }}/airflow.cfg /srv/{{ dockname }}/airflow.cfg
```

Comment:
`[copy] module copies a file on the local box to remote locations`. This task extracts shipped airflow.cfg from docker image and that happens on server side.